Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -623,6 +623,11 @@ public StyleNano(IResourceCache resCache) : base(resCache)
                     .Class(ButtonOpenLeft)
                     .Prop(ContainerButton.StylePropertyStyleBox, BaseButtonOpenLeft),
 
+                Element<TextureButton>().Class(ContainerButton.StyleClassButton)
+                    .Class(ButtonOpenLeft)
+                    .Prop(ContainerButton.StylePropertyStyleBox, BaseButtonOpenLeft),
+
```

Comment:
Is this needed? TextureButton with "ButtonOpenLeft" isn't used anywhere in the diff and regular `Buttons` are subclasses of ContainerButton.

Also, I don't think TextureButton even has a `StylePropertyStyleBox` (it's from ContainerButton), so this code wouldn't do anything.

```suggestion
```

Reply:
Yes, whoops - leftover from something else I was trying. Thanks.